addons/l10n_latam_base/models/res_partner.py

```diff
@@ -0,0 +1,28 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_latam_identification_type_id = fields.Many2one('l10n_latam.identification.type',
+        string="Identification Type", index=True, auto_join=True,
+        default=lambda self: self.env.ref('l10n_latam_base.it_vat', raise_if_not_found=False),
+        help="The type of identification",)
+    vat = fields.Char(string='Identification Number', help="Identification Number for selected type.")
```

Comment:
Why not as it is? When this module is installed the vat field is in concept not a vat number is a identification number

IMHO i think is better in the model because if we are going to search by the field or export records, would be good to have the Identification Number instead of VAT string